ams/server/src/main/java/com/netease/arctic/server/table/TableService.java

```diff
@@ -64,18 +64,25 @@ public interface TableService extends CatalogService, TableManager {
   List<String> listDatabases(String catalogName);
 
   /**
-   * load table identifiers
+   * load all table identifiers from system database
    *
-   * @return TableIdentifier list
+   * @return {@link ServerTableIdentifier} list
    */
-  List<ServerTableIdentifier> listTables();
+  List<ServerTableIdentifier> listSystemTables();
```

Comment:
Is `listManagedTables` better?
`Managed tables` means the tables which are managed by AMS, AMS will watch their change and make them health.